pyproject.toml

```diff
@@ -27,8 +27,10 @@ exclude = ["test/*", "examples/*", "docs/*"]
 
 [tool.poetry.dependencies]
 python = "^3.11"
+docstring-parser = "^0.15"
```

Comment:
This and jsonargparse should probably be within its own group. Maybe becoming optional later or even now, what do you think @opcode81 ?

Tianshou doesn't need these dependencies, but they are needed to run the examples. I guess it's a bit weird to have to install an extra in order to run examples. On the other side, we should avoid adding any unnecessary requirements to the core package, python package conflicts can be a real PITA

Reply:
If we want it to be pip-installable, it should be an extra (for example 'high-level-api'). A group would only be installable using poetry

Reply:
I guess an extra called `argparse` would make sense. You'd need to very visibly document that this extra is required to run the high-level-api examples